packages/core/src/compatibility/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+export * from "./browser";
```

Comment:
this index isn't useful. why not move `browser.ts` to `utils/browser.ts` instead of adding a new directory?